icu4c/source/common/rbbisetb.cpp

```diff
@@ -46,6 +45,7 @@
 
 U_NAMESPACE_BEGIN
 
+const int32_t kCharCategoriesThresholdFor8BitsTrie = 127;
```

Comment:
"threshold" is ambiguous (unclear whether it's the max or one higher); I would use "max" in the name, e.g. kMaxCharCategoriesFor8BitsTrie

Reply:
I agree with this suggestion.